core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/host/ClusterHostServiceRunner.java

```diff
@@ -582,6 +587,10 @@ private void saveGatewayPillar(GatewayConfig gatewayConfig, Cluster cluster,
         gateway.put("password", cluster.getPassword());
         gateway.put("enable_knox_ranger_authorizer", enableKnoxRangerAuthorizer);
 
+        String userCrn = stackService.get(stack.getId()).getCreator().getUserCrn();
+        final boolean ccmV2Enabled = entitlementService.ccmV2Enabled(Crn.safeFromString(userCrn).getAccountId());
```

Comment:
this must come from the tunnel and not from the entitlement. also final is not necessary

Reply:
Fixed, thank you!